src/pages/ExcludeWordsStep2.js

```diff
@@ -0,0 +1,96 @@
+import { isSupportedBrowser } from "../utils/misc/browserDetection";
+
+import useExcludeInterest from "../hooks/useExcludeInterest";
+import useFormField from "../hooks/useFormField";
+
+import InfoPage from "./info_page_shared/InfoPage";
+import Header from "./info_page_shared/Header";
+import Heading from "./info_page_shared/Heading";
+import Main from "./info_page_shared/Main";
+import Form from "./info_page_shared/Form";
+import Button from "./info_page_shared/Button";
+import Footer from "./info_page_shared/Footer";
+import ButtonContainer from "./info_page_shared/ButtonContainer";
+import ArrowForwardRoundedIcon from "@mui/icons-material/ArrowForwardRounded";
+import AddRoundedIcon from "@mui/icons-material/AddRounded";
+import HighlightOffRoundedIcon from "@mui/icons-material/HighlightOffRounded";
+
+import InputField from "./info_page_shared/InputField";
+import Tag from "./info_page_shared/Tag";
+import TagContainer from "./info_page_shared/TagContainer";
+
+import redirect from "../utils/routing/routing";
+import strings from "../i18n/definitions";
+
+export default function ExcludeWordsStep2({ api, hasExtension }) {
+  const {
+    subscribedSearchFilters,
+    subscribeToSearchFilter,
+    removeSearchFilter,
+  } = useExcludeInterest(api);
+
+  const [excludedWords, handleExcludedWordsChange, resetExcludedWords] =
+    useFormField("");
+
+  function getLinkToNextPage() {
+    if (isSupportedBrowser() && hasExtension === false) {
+      return "/install_extension";
+    } else return "/articles";
+  }
+
+  function handleAddNewSearchFilter(e) {
+    e.preventDefault();
+    if (excludedWords) {
+      subscribeToSearchFilter(excludedWords);
```

Comment:
**Discussion point:** Should we allow users to insert multiple times the same word as a keyword? I'd say we should check if the word is already part of the excludedWords and if so we don't add it again.

Backend: We are currently having duplication where the Search keyword is unique for all users, even if they have the same keyword.

Reply:
Good point! We should definitely implement these restrictions in further iterations.

Reply:
Let us open issues so we don't forget about this!